style/icons/shield40_us_mo_3.svg

```diff
@@ -0,0 +1,4 @@
+ <path style="fill:#fff" d="m 17.802726,18.999599 c -0.302184,0 -0.03819,-0.374013 0.313485,-0.869615 0.504114,-0.663209 0.901142,-1.683622 0.309213,-1.701756 -0.490477,0 -0.591231,0 -7.318716,0 -4.1823671,0 -6.923318,0 -7.3111753,0 -0.3878574,0 -0.3878574,0 -0.3915392,-0.284295 -0.00393,-0.284295 0.00393,-0.08758 0.00393,-4.556922 V 7.2943268 L 3.2506041,6.9353054 C 3.1618477,6.7326582 3.0203874,6.5178561 2.9637463,6.4499732 2.8766591,6.3455765 2.8607733,6.2111853 2.8607733,5.5775004 V 4.8284322 L 2.3756374,4.0674644 C 1.8496677,3.2424624 1.6282187,2.8151949 1.5593245,2.4924714 1.5350736,2.3787081 1.3971282,2.100942 1.2528795,1.8752221 0.94306686,1.3904495 1.2048686,1.0000001 1.4976762,1.0000001 c 3.5351238,0 11.5799328,0 16.9305758,0 0.138093,0 0.455102,0.1125949 0.497458,0.2665832 0.12418,0.4513908 -0.09953,0.5067555 0.116767,0.8736118 0.270854,0.459373 0.09563,0.5731951 0.291404,0.8120616 0.117828,0.143797 0.350381,0.4598148 0.51677,0.7022747 0.166389,0.2424599 0.396998,0.5635046 0.51247,0.713438 0.232111,0.3014083 0.417842,0.7020587 0.51244,1.1054288 0.05694,0.2426661 0.246102,0.5414431 0.567441,0.8961347 0.174735,0.1928781 0.664976,0.9680724 0.80677,1.2757054 0.106822,0.2317876 0.118937,0.3915687 0.131947,1.7401447 0.0088,0.877136 0.0431,1.617859 0.08453,1.809736 0.08944,0.413875 0.436369,1.200714 0.623297,1.413808 0.118241,0.134794 0.141736,0.213948 0.132222,0.4455 -0.0088,0.205072 0.02386,0.343773 0.118741,0.510447 0.383036,0.672683 0.495288,0.921573 0.528689,1.172281 0.05223,0.392167 -4.42e-4,0.545174 -0.327662,0.951676 -0.158819,0.197296 -0.317255,0.447503 -0.35209,0.556013 -0.150571,0.469064 -0.389791,0.774802 -0.788557,1.007797 -0.225572,0.131799 -0.412019,0.634176 -0.521679,0.856832 -0.05979,0.121343 -0.189923,0.271021 -0.192259,0.49143 -0.0049,0.427356 0.0375,0.398696 -0.508641,0.398696 -1.114108,0 -1.756138,0 -3.375333,0 z"/>
```

Comment:
It might be an optical illusion, but it seems like this shield is a horizontally stretched variation of the two-digit shield. On signs and maps, it’s more common to elongate the straight west–east portions of the state shape, leaving the bootheel and Mississippi River at their usual widths and angles. That could give us a little extra horizontal space to squeeze the digits into.

Reply:
![Screenshot from 2022-04-18 09-07-18](https://user-images.githubusercontent.com/1732117/163812724-4cf1b981-1934-4163-87d1-fee1d7c11c9c.png)
![Screenshot from 2022-04-18 09-07-32](https://user-images.githubusercontent.com/1732117/163812727-d3dd1415-0ce5-48f4-8baa-68e0d9a0ac5e.png)

Replaced the 3-digit variant with a modified 2-digit variant with elongated straight lines.